Library/Formula/seafile-client.rb

```diff
@@ -0,0 +1,42 @@
+class SeafileClient < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/seafile-client/archive/v4.1.3.tar.gz"
+  sha1 "52a0a4f78a3f73b3d3902502c2ce8989e79a748b"
+
+  head "https://github.com/haiwen/seafile-client.git", :branch => "4.1"
+
+  depends_on MinimumMacOSRequirement => :lion
```

Comment:
Yeah, Seafile Client doesn't support any OS X older than lion since version 4.0. It uses some features that don't come along with OS X 10.6.